pkg/agent/route/route_linux.go

```diff
@@ -1245,6 +1258,38 @@ func (c *Client) initWireguard() {
 	}
 }
 
+func (c *Client) initNodeLatency() {
+	gateway := "antrea-gw0"
```

Comment:
this is configurable in theory so you should not hardcode it

the gateway name should be available in the `nodeConfig` field, please take a look